recipes/civetweb/all/conanfile.py

```diff
@@ -64,6 +80,8 @@ def configure(self):
     def requirements(self):
         if self.options.with_ssl:
             self.requires("openssl/1.1.1m")
+        if self.options.with_zlib:
+            self.requires("zlib/1.2.11")
```

Comment:
didn't check other options, but this one didn't exist before 1.15

Reply:
Oh, you're right!
I didn't think about incorporating this dimension before. I'll do the fixes.

Reply:
Fixed in [ad68836](https://github.com/conan-io/conan-center-index/pull/9303/commits/ad688360d18a6befac9b2358336364e3c17cf2b0).
I went back in time in all the published versions on the Conan Center, and luckily `zlib` was the only one changing.

Thanks for the help!